assets/pages/support-landing/components/patronsEvents.js

```diff
@@ -0,0 +1,25 @@
+// @flow
```

Comment:
Is it possible to delete this file? It looks like we are just populating a prop from the state and nothing else.

Reply:
I've described why I did this in the [description](https://github.com/guardian/support-frontend/pull/468#issue-291560568).

Reply:
I am not entirely convinced that this a good approach. We tried this before in the past when we added the paymentMethodContainer which we finally deleted. Since our state is more ordered now (there is the common and page structure in place), I would consider building a connected shared component, with a good documentation (add a readme in that folder) and in that way, we will avoid creating empty components, i.e. boilerplate.

Reply:
Is it necessary to build `patronsEvents.js` as a shared component? If we need to share it across localised versions of the same landing page, we could put it in a component folder inside a page folder. What do you think?